### PR TITLE
reset player weak reference to null on release

### PR DESCRIPTION
**Pull request by wtao-mux**

### CODE REVIEW

File: MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java [hunk 1]
```diff
@@ -110,13 +111,13 @@ public void setStreamType(int type) {
 
     @Override
     public void dispatch(IEvent event) {
-        if (player.get () != null && muxStats != null)
+        if (player != null && player.get () != null && muxStats != null)
```
Comment (jsanford8) [inline MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java]: this was in there before, but while we're in here, change `player.get ()` to `player.get()`

File: MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java [hunk 2]
```diff
@@ -485,15 +486,15 @@ public BandwidthMetric currentBandwidthMetric() {
         }
 
         public void onLoadError(DataSpec dataSpec, int dataType, IOException e) {
-            if (player.get () == null || muxStats == null || currentBandwidthMetric() == null) {
+            if (player == null || player.get () == null || muxStats == null || currentBandwidthMetric() == null) {
```
Comment (jsanford8) [inline MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java]: same as above.

**Review by jsanford8 — COMMENTED** [review]
looks good, just a super minor nit on spacing

**Comment by wtao-mux** [comment]
yup. sounds good. i have the spaces removed

**Review by jsanford8 — APPROVED** [review]

**Pull request merged**